ci-operator/step-registry/upi/libvirt/deprovision/upi-libvirt-deprovision-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+LIBVIRT_DOMAIN_NAME_SUFFIX="libvirt-s390x-amd64-0-0-ci"
+
+export LIBVIRT_DEFAULT_URI="qemu+tcp://lnxocp10:16509/system"
```

Comment:
This probably should be part of cluster profile setting? Can be fixed later

Reply:
Good point. The setup is rather static right now but it makes total sense to refactor this when we have multiple clusters where this varies in the future (same for CLUSTER_DOMAIN and for the XML templates that are currently embedded in the files).